great_tables/_export.py

```diff
@@ -559,3 +559,47 @@ def _dump_debug_screenshot(driver, path):
         "document.getElementsByTagName('table')[0].style.border = '3px solid green'; "
     )
     driver.save_screenshot(path)
+
+
+def write_raw_html(
+    gt: GT,
+    filename: str | Path | None = None,
+    encoding: str = "utf-8",
+    newline: str | None = None,
+    make_page: bool = False,
+    all_important: bool = False,
+) -> str | None:
```

Comment:
@rich-iannone you mentioned type stability, but this does not have the stability you mentioned. Is this what you want in the interface?

Reply:
Thanks for noting this. @jrycw could the signature be this instead:

```
def write_raw_html(
     gt: GT,
     filename: str | Path,
     encoding: str = "utf-8",
     newline: str | None = None,
     make_page: bool = False,
     all_important: bool = False,
 ) -> None:
```

Then these lines would need to be removed:

```
if filename is None:
         return html_content
```

Since we are requiring a `filename=` value, the last sentence of `filename` parameter description in the docstring needs to removed. Also, the description in Returns needs to change to fit a return of `None`.

Reply:
I'm down for whatever seems reasonable -- in this case it sounds like the intended spec is...

**Polars**:
  - `to_*()` methods return objects.
  - `write_*()` methods return None if writing to file, string otherwise.

**Great Tables**:
  - `as_*()` methods return objects.
  - `write_*()` methods always return None (and filename is mandatory).